### DISCUSSION TITLE
xbrl_data.presentation data access

### DISCUSSION THREAD

**Discussion by webbsledge** [dgunning/edgartools#155] [General]
Is there any way to access the data contained within each item that has been parsed into the `xbrl_data.presentation` tree? This seems like a really powerful way to understand the XBRL data structure, but I can't seem to figure out how to actually see what's in each tree branch.

<img width="1159" alt="Screenshot 2024-11-23 at 1 26 13 PM" src="https://github.com/user-attachments/assets/8645afbf-21f1-47d6-899a-5a920fc63edb">

**Comment by dgunning** [comment]
Great question. Would be worth adding this feature. This depends of course on how people would actually use this feature so would be nice to get some usage examples
